test/fixtures/templates.yml

```diff
@@ -86,6 +86,22 @@ pxegrub:
    locked: false
    type: ProvisioningTemplate
 
+pxegrub_default:
+   name: PXEGrub global default
+   template: "FOO"
+   template_kind: pxegrub
+   operatingsystems: centos5_3, redhat
+   locked: false
+   type: ProvisioningTemplate
+
+pxegrub2_default:
```

Comment:
Since there are now two default templates (for two different `TemplateKind::PXE`), calling `build_pxe_default` will attempt to render two instead of one. https://github.com/theforeman/foreman/blob/develop/test/models/provisioning_template_test.rb#L219 the expectation should be expected twice